### PR TITLE
Add classes tests

### PR DESCRIPTION
**Pull request by FalseR20**

### CODE REVIEW

File: tests/test_classes.py [hunk 1]
```diff
@@ -0,0 +1,119 @@
...
+
+        class AgentClassic(ScAgentClassic):
+            def on_event(self, init_element: ScAddr, init_edge: ScAddr, action_element: ScAddr) -> ScResult:
+                if not self._confirm_action_class(action_element):
+                    return ScResult.SKIP
+                finish_action_with_status(action_element, True)
+                return ScResult.OK
+
+        agent = Agent(event_class=Agent.ACTION_CLASS_NAME, event_type=ScEventType.ADD_OUTGOING_EDGE)
+        agent_classic = AgentClassic(classic_class_name := "classic")
```
Comment (AlexandrZagorskiy) [inline tests/test_classes.py:32]: Why ScAgent can be created without passed identifier, but ScAgentClassic used with it? Can we create ScAgentClassic objects without passing arguments?

File: tests/test_classes.py [hunk 2]
```diff
@@ -0,0 +1,119 @@
...
+        agent1 = TestAgent("agent1", ScEventType.ADD_OUTGOING_EDGE)
+        agent2 = TestAgent("agent2", ScEventType.ADD_OUTGOING_EDGE)
+        agent3 = TestAgent("agent3", ScEventType.ADD_OUTGOING_EDGE)
+
+        module1 = ScModule(agent1)
+        module2 = ScModule(agent2)
+        server.add_modules(module1)
+        module1.add_agent(agent3)
+        with server.register_modules():
+            module1.add_agent(agent2)  # Cannot add agent to module in register time
```
Comment (AlexandrZagorskiy) [inline tests/test_classes.py:76]: what does this comment mean for developers? Is it an error, or such behavior is expected?

File: tests/test_classes.py [hunk 3]
```diff
@@ -0,0 +1,119 @@
...
+            self.assertTrue(is_executing_successful())
+        self.assertFalse(is_executing_successful())
+
+        server.register_modules()
+        self.assertTrue(is_executing_successful())
+        server.remove_modules(module)
+        self.assertFalse(is_executing_successful())
+        server.add_modules(module)
+        server.unregister_modules()
+        self.assertFalse(is_executing_successful())
```
Comment (AlexandrZagorskiy) [inline tests/test_classes.py:119]: can we add tests with delayed called agent?

**Review by AlexandrZagorskiy — CHANGES_REQUESTED** [review]

File: tests/test_classes.py (hunk 2 above)
Comment (FalseR20) [reply tests/test_classes.py:76]: This line do nothing and I wated mark it. But yeah, better to remove

File: tests/test_classes.py [hunk 4]
```diff
@@ -0,0 +1,119 @@
...
+from sc_kpm.utils.action_utils import execute_agent, finish_action_with_status
+from tests.common_tests import BaseTestCase, server
+
+WAIT_TIME = 1
+
+
+class CommonTests(BaseTestCase):
+    def test_sc_agents(self):
+        class Agent(ScAgent):
+            ACTION_CLASS_NAME = "Stalin_3000"
```
[the remark this answers is not in the document]
Comment (FalseR20) [reply tests/test_classes.py]: Done

File: tests/test_classes.py (hunk 1 above)
Comment (FalseR20) [reply tests/test_classes.py]: Done

File: tests/test_classes.py (hunk 3 above)
Comment (FalseR20) [reply tests/test_classes.py:127]: Done

**Review by AlexandrZagorskiy — APPROVED** [review]

**Pull request merged**